README.md

```diff
@@ -123,6 +123,54 @@ If you want to use Isaac Gym, use python3.8 (not python3.9)
 5. nvidia-smi` to check that you have NVIDIA drivers and proper cuda installations.
 6. (Assuming you already have mlflow running in a different terminal), Run `python -m main services/environment=ant`
 
+## Petting Zoo
+### Installation
+  1. Clone this repository
+      ```sh
+      git clone https://github.com/cogment/cogment-verse.git
+      ```
+  2. Create and activate the virtual python environment
+      ```sh
+      python -m venv .venv
+      source .venv/bin/activate
+      ```
+  3. This step is only required for Apple silicon-based computers (e.g., M1&2 chips). Clone [Multi-Agent-ALE](https://github.com/Farama-Foundation/Multi-Agent-ALE) repository
+      ```sh
+      git clone https://github.com/Farama-Foundation/Multi-Agent-ALE.git
```

Comment:
The instructions should target a directory that is not versioned.

Reply:
I don't fully understand what you meant. However, I will move this stuff to `development_setup.md` for now. I already fixed this issue for Farama last December but they haven't released the new version yet.